### PR TITLE
SEAR-4 Facility Profile - Add selectors pattern

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to (link)?
- https://osi-cwds.atlassian.net/browse/SEAR-4

#### Brief feature description
- Add Selectors to remove business logic from components.

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
- n/a

### CODE REVIEW

**Comment by Nasir2712** [comment]
jenkins, retest this please

**Comment by Nasir2712** [comment]
jenkins, retest this please

File: app/javascript/facility/facilityTypeDetails.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+const FacilityTypeDetails = ({
```
Comment (ratneshraval) [inline app/javascript/facility/facilityTypeDetails.js]: 👍

File: app/javascript/selectors/facilityDataSelectors.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
+import {
```
Comment (ratneshraval) [inline app/javascript/selectors/facilityDataSelectors.js]: What's the thought on keeping this file here vs `javascript/facility` or `javascript/facility/selectors`
Eventually less looking around for where the file is.

Comment (Nasir2712) [reply app/javascript/selectors/facilityDataSelectors.js]: so the thought process behind this folder structure is similar to actions and reducers, so selectors and container will also be in separate folders so one facility file will have facility action, facility reducer, facility container and facility selector.

**Comment by Nasir2712** [comment]
jenkins, retest this please

**Pull request merged**